activerecord/lib/active_record/connection_adapters/mysql/schema_definitions.rb

```diff
@@ -7,6 +7,10 @@ def primary_key(name, type = :primary_key, **options)
           super
         end
 
+        def blob(*args, **options)
```

Comment:
Do we actually need this? It looks like `binary` with with the right limit should be fine.